Header header

float32[] internal_load_cell_force
float32[] external_load_cell_force
